vignettes/vignette.Rnw

```diff
@@ -457,26 +457,35 @@ oncoprint(dataset, group.samples = as.stages(dataset))
 
 TRONCO provides functions to visualize groups of events, which in this case are called pathways -- though this could be any group that one would like to define. Aggregation happens with the same rational as the \Rfunction{as.alterations} function, namely by merging the events in the group.
 
-We make an example of a pathway called {\tt MyPATHWAY} involving genes SETBP1, EZH2 and WT1; we want it to be colored in red, and we want to have the genotype of each event to be maintened in the dataset. We proceed as follows  (R's output is omitted).
-
+We make an example of human pathways from \href{https://wikipathways.org}{WikiPathways} involving genes SETBP1, EZH2, TET2, IRAK4, SETBP1 and KIT1. We can query WikiPathways and collect HGNC gene symbols and titles for pathways of interest as follows.  (R's output is omitted).
 
 <<pathway>>=
+library(rWikiPathways)
```

Comment:
The library should be added to the `Suggest` section of the `DESCRIPTION` file. No need to do things like:
```
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("rWikiPathways")
```